CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
 add_subdirectory(el)
-add_subdirectory(sc)
\ No newline at end of file
+add_subdirectory(sc)
+install(DIRECTORY HelpSource
+        DESTINATION share/SuperCollider/Extensions/scide_scel/)
```

Comment:
i think this needs to change for macOS, but i see `sc/CMakeLists.txt` does this too, so it's already broken there. probably worth logging an issue for that and resolving it at some point.

Reply:
actually, it appears there is already a related issue - https://github.com/supercollider/scel/issues/12